src/components/Pagination/index.jsx

```diff
@@ -60,80 +61,82 @@ const Pagination = ({
   const shouldDisplayPrev = currentPageIndex === 1;
   const shouldDisplayNext = currentPageIndex === totalPages;
   return (
-    <div className="pagination pagination-container">
-      <a
-        className={`btn btn-secondary btn-small pagination-prev ${
-          shouldDisplayPrev ? "is-disabled" : ""
-        }`}
-        onClick={handlePrevClicked}
-        onKeyDown={handlePrevClicked}
-        role="button"
-        tabIndex="0"
-      >
-        Prev
-      </a>
-      {shouldDisplayFirstPage && (
-        <span>
-          <a
-            className="pagination-link"
-            onClick={updatePageIndex}
-            onKeyDown={updatePageIndex}
-            value={1}
-            role="button"
-            tabIndex="0"
-          >
-            {1}
-          </a>
-          <a className="pagination-ellipses">&hellip;</a>
-        </span>
-      )}
-      <div className="pagination-links">
-        {Array(endPage - startPage + 1)
-          .fill()
-          .map((_, i) => startPage + i)
-          .map(number => (
-            <a
-              key={number}
-              className={cn("pagination-link", {
-                "is-active": number === currentPageIndex,
-              })}
-              onClick={updatePageIndex}
-              onKeyDown={updatePageIndex}
-              value={number}
-              role="button"
-              tabIndex="0"
-            >
-              {number}
-            </a>
-          ))}
-        {shouldDisplayLastPage && (
+    <Column width={10} offset={2}>
+      <div className="pagination pagination-container">
+        <a
+          className={`btn btn-secondary btn-small pagination-prev ${
+            shouldDisplayPrev ? "is-invisible" : ""
+          }`}
+          onClick={handlePrevClicked}
+          onKeyDown={handlePrevClicked}
+          role="button"
+          tabIndex="0"
+        >
+          Prev
+        </a>
+        {shouldDisplayFirstPage && (
           <span>
-            <a className="pagination-ellipses">&hellip;</a>
             <a
               className="pagination-link"
               onClick={updatePageIndex}
               onKeyDown={updatePageIndex}
-              value={totalPages}
+              value={1}
               role="button"
               tabIndex="0"
             >
-              {totalPages}
+              {1}
             </a>
+            <a className="pagination-ellipses">&hellip;</a>
           </span>
         )}
+        <div className="pagination-links">
+          {Array(endPage - startPage + 1)
```

Comment:
Let's move this inside of a function and also set the calculation ```endPage - startPage + 1``` to a readable variable name to show more intent so the JSX will have one function call name like ```showPaginationLinks()``` that encapsulates all this logic. We also don't need to do two maps but one map and set startPage + 1 to a variable like currentPaginationNum = startPage + i and return the <a> tags